pkg/hive/hive.go

```diff
@@ -97,13 +95,14 @@ func (s *Service) sendPeers(ctx context.Context, peer swarm.Address, peers []swa
 	for _, p := range peers {
 		addr, err := s.addressBook.Get(p)
 		if err != nil {
-			if errors.Is(err, storage.ErrNotFound) {
-				s.logger.Debugf("Peer not found %s", p)
-				continue
-			}
 			return err
 		}
 
+		if addr == nil {
+			s.logger.Debugf("Peer not found %s", p)
```

Comment:
Aaaand, this is the comment that was not submitted. I think that we can provide a bit more information in this message like "hive: peer not found in address book: %s".

Reply:
I agree! Changed to a bit more info. Please LTMWYT